src/Umbraco.Core/IO/IOHelper.cs

```diff
@@ -298,7 +294,7 @@ public string Root
             {
                 if (_root != null) return _root;
 
-                var appPath = HttpRuntime.AppDomainAppVirtualPath;
+                var appPath = HostingEnvironment.ApplicationVirtualPath;
```

Comment:
It should, but I need to figure out IOHelpers usages of System.Web in a sepeate task.. 

IHostingEnvironment and IIOHelper is very closely related, and We need to avoid a circular dependency between these. In the end some methods will probably move from IOHelper to HostingEnvironment, like MapPath, etc